configuration/src/main/resources/db/migration/V20200416__init-schema.sql

```diff
@@ -0,0 +1,134 @@
+CREATE TABLE IF NOT EXISTS Position
```

Comment:
not 'position' ?

Reply:
do you mean we need to lowercase all table names? no problem